motion/ruby_motion_query/device.rb

```diff
@@ -92,7 +92,7 @@ def iphone?
       def simulator?
         if @_simulator.nil?
           @_simulator = if ios_at_least?(9.0)
-            !NSBundle.mainBundle.bundlePath.start_with?('/var/')
+            !(NSBundle.mainBundle.bundlePath =~ /Library\/Developer/i).nil?
```

Comment:
Interesting, at one time this was not the case (it did not report that way which is why it got changed in the first place, they must have fixed that...maybe?

see this commit when this change was introduced: cbf2c117c55e7c71ae74412e06f9116a00d6ba72